### PR TITLE
Put nvram into XDG location, improve FreeBSD compatibility

### PR DESCRIPTION
**Pull request by clausecker**
As discussed on IRC.

### CODE REVIEW

File: vt100/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 all: vt100
-CFLAGS=-O3
+CFLAGS?=-O3
```
Comment (larsbrinkhoff) [inline vt100/Makefile:2]: Is this related to -DSHADERDIR?

Comment (clausecker) [reply vt100/Makefile:2]: Yes.  This allows `CFLAGS` to be overwritten from the environment.  It is not strictly needed; I can also maintain this change locally as a FreeBSD only patch.

Comment (larsbrinkhoff) [reply vt100/Makefile:2]: Tanks, just checking.

**Comment by larsbrinkhoff** [comment]
Merged, thanks!
